This patch circumvents an error with 'const char *' to 'char *' conversion,
see http://gnats.netbsd.org/45562 .

diff -u a/xpdf/XPDFViewer.cc b/xpdf/XPDFViewer.cc
--- a/xpdf/XPDFViewer.cc        2011-08-15 23:08:53.000000000 +0200
+++ b/xpdf/XPDFViewer.cc        2013-03-31 15:13:21.000000000 +0200
@@ -1803,7 +1803,7 @@
   menuPane = XmCreatePulldownMenu(toolBar, "zoomMenuPane", args, n);
   for (i = 0; i < nZoomMenuItems; ++i) {
     n = 0;
-    s = XmStringCreateLocalized(zoomMenuInfo[i].label);
+    s = XmStringCreateLocalized((char *) zoomMenuInfo[i].label);
     XtSetArg(args[n], XmNlabelString, s); ++n;
     XtSetArg(args[n], XmNuserData, (XtPointer)i); ++n;
     sprintf(buf, "zoom%d", i);
